robot/web/web.go

```diff
@@ -43,7 +43,13 @@ import (
 )
 
 // SubtypeName is a constant that identifies the internal web resource subtype string.
-const SubtypeName = "web"
+const (
+	SubtypeName   = "web"
+	TCPParentPort = 14998
```

Comment:
What happens when this port is occupied?

Reply:
TCP sockets support breaks

follow-ups ticket [APP-7240](https://viam.atlassian.net/browse/APP-7240) will change this to zero to auto-assign